Changelog.md

```diff
@@ -1,5 +1,27 @@
 ## Ignition Sensors 3
 
+### Ignition Sensors 3.X.X (20XX-XX-XX)
+
+### Ignition Sensors 3.1.0 (2020-09-09)
```

Comment:
update date?

Reply:
Either I'm living in the future, or I didn't expect this PR to be merged so soon :sweat_smile: 0c312ef9fa26d8be2ff293f0e8d8f71dc0d5fe74